naurffxiv/src/app/navigation.js

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import { Container } from "@mui/material";
+import { Grid } from '@mui/material';
+// import MenuIcon from '@mui/icons-material/Menu';
+
+export default function NavBar() {
+

```

Comment:
can we put two ` // TODO:`s here, one being replacing the href links to their respective pages and the other to replace the NAUR with the favicon